src/test/java/lottogame/domain/WinningLottosTest.java

```diff
@@ -0,0 +1,57 @@
+package lottogame.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class WinningLottosTest {
+
+    @Test
+    void 생성() {
+        WinningLottos winningLotto = LottoFactory.createWinningLotto(List.of(1, 2, 3, 4, 5, 6),  7);
+        int expected = 6;
+
+        assertThat(winningLotto.size()).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @MethodSource("lottoAndRank")
+    void 로또_당첨(Lottos lottos, Rank rank) {
+        WinningLottos winnerLotto = new WinningLottos(List.of(1, 4, 5, 6, 7, 8), 9);
+        System.out.println("winnerLotto.getRank(lottos) = " + winnerLotto.checkRank(lottos));
+        assertThat(winnerLotto.checkRank(lottos)).isEqualTo(rank);
+    }
+
+    @ParameterizedTest
+    @MethodSource("lottoAndPrize")
+    void 로또_당첨_금액(Lottos lottos, Number prize) {
+        WinningLottos winnerLotto = new WinningLottos(List.of(1, 4, 5, 6, 7, 8), 9);
+        assertThat(winnerLotto.checkPrize(lottos)).isEqualTo(prize);
```

Comment:
```suggestion
        assertThat(winnerLotto.상금계산해줘(lottos)).isEqualTo(prize);
        assertThat(winnerLotto.calculatePrize(lottos)).isEqualTo(prize);
```

저는 이런 관점이 떠오르네요~
`get-` 보다는 명확한 메시지를 보내기의 의도였습니다 ㅎ